js/script.js

```diff
@@ -72,25 +128,98 @@
       filterState = state || filterState;
       return tasks.filter(filters[filterState]);
     }
+  })
+
+  app.service("PageService", function(TaskResource){
+
+    var limit = 10, totalCount, currentPage = 0, orderBy, filterBy
```

Comment:
My rule is as-follows: I’ll put multiple comma-separated declarations in a single var statement, but only if they don’t have assignments. Assignments must be in their own var statement.